### PR TITLE
Add CITATION.cff file

### LINKED ISSUE #83: Add citation information through CITATION.cff
No citation information is currently available for this repository

### PR DESCRIPTION
**Pull request by sjvrijn**
Fixes #83

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    name-particle: Lo
+    family-names: Cascio
+    email: ermanno.locascio@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0002-7257-8034'
+repository-code: 'https://github.com/ESI-FAR/INA-tool'
+url: 'https://inaeditor.tbm.tudelft.nl'
+keywords:
+  - Institutional Network Analysis
+  - INA
```
Comment (suvayu) [inline CITATION.cff:27]:
```suggestion
```
IMO "INA" too much of a generic term, also a bit redundant here.

That said, adding a more generic term could be useful. Don't have a good suggestion, I'm thinking of "social graph" but that probably invokes "social media" in everyone's mind. Maybe we can ask Amineh/Renske for suggestions, and add it later?

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - Institutional Grammer
```
Comment (suvayu) [inline CITATION.cff:28]:
```suggestion
  - Institutional Grammar
```

**Review by suvayu — CHANGES_REQUESTED** [review]
Is there some way to also cite the idea of Institutional grammar itself? Maybe Amineh has a paper, or there are older papers from the literature she can suggest?

We can of course create an issue and add this later (don't want hold up this PR for a response).

**Review by suvayu — APPROVED** [review]

**Comment by sjvrijn** [comment]
> Is there some way to also cite the idea of Institutional grammar itself? Maybe Amineh has a paper, or there are older papers from the literature she can suggest?
> 
> We can of course create an issue and add this later (don't want hold up this PR for a response).

Good point! I've noted that in a new issue #87

**Pull request merged**